lib/Zonemaster/LDNS/RR/CDNSKEY.pm

```diff
@@ -0,0 +1,22 @@
+package Zonemaster::LDNS::RR::CDNSKEY;
+
+use strict;
+use warnings;
+
+use parent 'Zonemaster::LDNS::RR';
+
+1;
+
+=head1 NAME
+
+Zonemaster::LDNS::RR::CDNSKEY - Type CDS record
+
+=head1 DESCRIPTION
+
+A subclass of L<Zonemaster::LDNS::RR>, so it has all the methods of that class available in addition to the ones documented here.
+
+=head1 METHODS
```

Comment:
A CDNSKEY RR has the same structure of the RDATA as a DNSKEY. In the same way as CDS RR has the same structure of the RDATA as DS (I saw that CDS.pm is also incomplete).

Reply:
That's a problem, but with an easy workaround (see in the next PR zonemaster/zonemaster-engine#896)